.changes/next-release/feature-retries-29845.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "feature",
+  "category": "retries",
+  "description": "Update the default retry mode from ``legacy`` to ``standard``. See `Boto3 Retries <https://boto3.amazonaws.com/v1/documentation/api/latest/guide/retries.html>`__ for more information about retry behavior and how to configure different retry options."
```

Comment:
I just realized this is in botocore and not boto3. The botocore commits get propagated to both Boto3 _and_ the AWSCLI. We should avoid pushing up Boto3 specific changelogs from this package.

If we're really concerned about pointing people to the guide, we can open a tracking issue and point to that for Botocore with all the information.